### PR TITLE
Added colorify and burn

### PR DESCRIPTION
**Pull request by Scientific-Guy**
Simple burn meme and colorify which is actually useless but ok

### CODE REVIEW

File: src/Canvas.js [hunk 1]
```diff
@@ -24,6 +24,27 @@ class Canvacord {
...
+    
+    /**
+     * colorify
+     * @param {Image} image to color
+     * @param {Hex} hex to color the image
+     * @returns {Promise<Buffer>}
+     * @example let img = await canva.colorify(img, hex);
+     * canva.write(img, "img.png");
+     */
+    static async colorify(image1, hex) {
```
Comment (abh80) [inline src/Canvas.js:36]: Looks OK , but one question the function takes only one `Image` so shouldn't the `image1` parameter should only be `image` and also here - 
```js
 if (!image1) throw new Error("first image was not provided!");
```
Using `first image` doesn't seem OK

**Review by abh80 — CHANGES_REQUESTED** [review]

**Comment by Snowflake107** [comment]
Please use `v5-alpha` branch

**Comment by Scientific-Guy** [comment]
:pensive: This is why i don't do pr coz i am noob myself

File: src/Canvas.js (hunk 1 above)
Comment (Scientific-Guy) [reply src/Canvas.js:36]: Well, yeah lmao i didn't noticed it!

**Pull request closed without merging**